cavefish-server/cavefish/core/src/Intent/Example/DSL.hs

```diff
@@ -40,6 +37,10 @@ import Ledger (
   cardanoPubKeyHash,
   pubKeyHash,
  )
+import Plutus.Script.Utils.Address (
+  ToAddress (toAddress),
+  ToPubKeyHash (toPubKeyHash),
+ )
 import WBPS.Core.Cardano.TxAbs (TxAbs (absFee, absMint, outputs, sigKeys, validityInterval))
 
 type ChangeDelta = Api.Value
```

Comment:
not part of this PR, but it doesn't seem we are using this anywhere. May we remove it?